README.md

```diff
@@ -185,6 +185,65 @@ refund = mollie_client.refunds.on(payment).create({
 })
 ```
 
+## Oauth2 ##
+
+Run the oauth2 examples.
+```
+FLASK_APP=examples/oauth/app.py \
+CLIENT_ID=your_client_id \
+CLIENT_SECRET=your_client_secret \
+REDIRECT_URI=https://your_domain.tld/callback \
+flask run
+```
+
+The Authorize endpoint is the endpoint on Mollie web site where the merchant logs in, 
+and grants authorization to your client application. 
+E.g. when the merchant clicks on the Connect with Mollie button, 
+you should redirect the merchant to the Authorize endpoint.
+
+The resource owner can then grant the authorization to your client application for the scopes you have requested.
+
+Mollie will then redirect the resource owner to the redirect_uri you have specified. 
+The redirect_uri will be appended with a code parameter, which will contain the auth token. 
+You should then exchange the auth token for an access token using the Tokens API.
+
+Intitializing via oauth2
+
+```python
+
+def get_token():
+    """A method you should supply to retrieve a stored token. Return json or None."""
+    ...
+
+def set_token(token):
+    """A method you should supply to permanently store the token (json)"""
+    ...
+
+
+mollie_client = Client()
+authorized, authorization_url = mollie_client.setup_oauth(
+        client_id,
+        client_secret,
+        redirect_uri,
+        scope,
+        get_token(),
```

Comment:
`set_token` should not have `()` here, you're passing the callable, not the result of the callable.